lemmy_db/src/schema.rs

```diff
@@ -0,0 +1,636 @@
+table! {
```

Comment:
This file needs to be removed, it gets auto-generated when you run diesel.

Reply:
fixed